src/plib/controllers/IndexController.php

```diff
@@ -11,6 +11,17 @@ class IndexController extends pm_Controller_Action
 
     protected $_accessLevel = ['admin', 'reseller', 'client'];
 
+    private function getActionUrl($controller, $action, array $params = [])
```

Comment:
you have only one controller - let's remove it from arguments

Reply:
OK